src/components/home/Home.jsx

```diff
@@ -10,24 +10,25 @@ import MusicPlayer from '../music/MusicPlayer';
 const Home = ({ route }) => {
   const { location } = route.params;
   const [token, setToken] = useState('');
-  const [userData, setUserData] = useState('');
+  const [userData, setUserData] = useState({});
```

Comment:
is there a reason to change initial value for userData? just curious

Reply:
also curious about this, since {} will be true for javascript, and '' is false

Reply:
revert it back, it wasnt necessary to make this change